src/services/TokenService.ts

```diff
@@ -0,0 +1,43 @@
+import axios from "axios"
+
+class TokenService {
+  baseURL: string
+  symbols: any[]
+  constructor() {
+    this.baseURL = "https://stage-auth.loom.games/wallet/tokens" // need to set base url for different env
+    this.symbols = []
+    this.init()
+  }
+
+  /**
+   * should call this method after service class created
+   */
+  async init() {
+    const result = await axios.get(this.baseURL)
+    this.symbols = result.data.tokens
+  }
+
+  setBaseURL(url: string) {
+    this.baseURL = url
+  }
+  /**
+   * 
+   * @param coinSymbol like BNB, LOOM, ETH
+   */
+  getTokenAddressBySymbol(coinSymbol: string) {
```

Comment:
we will also need the address on ethereum, so we should have an additional argument 
`chain: "plasma"|"ethereum"`